spec.md

```diff
@@ -1873,6 +1906,17 @@ message Snapshot {
   // `volume_content_source` in a `CreateVolumeRequest`. The default
   // value is false. This field is REQUIRED.
   bool ready_to_use = 5;
+
+  // The ID of the volume group snapshot that this snapshot is part of.
+  // It uniquely identifies the group snapshot on the storage system.
+  // This field is OPTIONAL.
+  // If this snapshot is a member of the volume group snapshot, the SP
+  // SHOULD provide the ID of the volume group snapshot in this field.
+  // If provided, CO MUST use this field to indicate that this snapshot
+  // is part of the specified group snapshot.
```

Comment:
nit: Add guidance for CO if not provided but is part of a group.

Reply:
Added this:
  // If not provided, CO SHALL not indicate this snapshot is part of
  // a group snapshot and allow it to be deleted separately.